### PR TITLE
Special report page

### PR DESCRIPTION
**Pull request by ironsidevsquincy**
Specific styling for special report stream page, e.g. header styling and 'more ons' at the bottom

screen shot 2015-07-31 at 16 12 40
...
screen shot 2015-07-31 at 16 12 48

### CODE REVIEW

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-07-31/15:31-133/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-07-31/15:31-133/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-07-31/16:10-136/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-07-31/16:10-136/successes/index.html)

File: views/special-reports.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 					<div data-o-grid-colspan="12 S10">
 						<h3 class="n-card__headline">
-							<a href="/stream/{{taxonomy}}Id/{{ id }}{{referrerTracking}}" data-trackable="headline">
+							<a href="/special-reports/{{id}}{{referrerTracking}}" data-trackable="headline">
```
Comment (matthew-andrews) [inline views/special-reports.html]: is it worth adding `302`s between flag on versus flag off?

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-03/08:43-144/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-03/08:43-144/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-03/09:04-147/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-03/09:04-147/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-03/11:29-151/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-03/11:29-151/successes/index.html)

**Comment by next-team** [comment]
I've spotted a [difference](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-03/11:40-154/failures/index.html) between your pull request and production — [see full output](https://s3-eu-west-1.amazonaws.com/ft-next-qa/image_diffs/stream-page/2015-08-03/11:40-154/successes/index.html)

**Pull request merged**